sycl/doc/extensions/supported/sycl_ext_intel_queue_immediate_command_list.asciidoc

```diff
@@ -0,0 +1,160 @@
+= sycl_ext_intel_queue_immediate_command_list
+
+:source-highlighter: coderay
+:coderay-linenums-mode: table
+
+// This section needs to be after the document title.
+:doctype: book
+:toc2:
+:toc: left
+:encoding: utf-8
+:lang: en
+:dpcpp: pass:[DPC++]
+
+// Set the default source code type in this document to C++,
+// for syntax highlighting purposes.  This is needed because
+// docbook uses c++ and html5 uses cpp.
+:language: {basebackend@docbook:c++:cpp}
+
+
+== Notice
+
+[%hardbreaks]
+Copyright (C) 2023-2023 Intel Corporation.  All rights reserved.
+
+Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are trademarks
+of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc. used by
+permission by Khronos.
+
+
+== Contact
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/intel/llvm/issues
+
+
+== Dependencies
+
+This extension is written against the SYCL 2020 revision 7 specification.  All
+references below to the "core SYCL specification" or to section numbers in the
+SYCL specification refer to that revision.
+
+
+== Status
+
+This extension is implemented and fully supported by {dpcpp}.
+
+However, the immediate command list feature (which is exposed by this extension)
+has been well-tested only on Intel (R) Data Center Max Series GPUs (aka PVC).
+Use of this extension to specify immediate command lists is not recommended
+for other Intel GPUs.
+
+
+== Backend support status
+
+The properties added by this extension are a hint, which all backends accept.
+However, in the current {dpcpp} implementation, the hint is only meaningful
+on the Level Zero backend.
+
+== Overview
+
+This extension enables specifying one of two submission modes for a queue.
+
+The property `no_immediate_command_list` specifies that submissions to a SYCL
+queue should be collected into groups before actual submission to the hardware.
+
+The property `immediate_command_list` specifies that submissions to a
+SYCL queue should be submitted to the hardware immediately.
+
+When the submission mode is left unspecified a platform-specific default
+is chosen by the implementation.
+
```

Comment:
This information is now all redundant, so delete these lines.  The properties are described below in the Specification section.

Reply:
OK